src/jsep.js

```diff
@@ -33,6 +34,8 @@
 		QUMARK_CODE = 63, // ?
 		SEMCOL_CODE = 59, // ;
 		COLON_CODE  = 58, // :
+		EQUAL_CODE  = 61, // =
```

Comment:
`EQUAL_CODE` is unused.

Reply:
I debated using a Map for the `expression_parsers` versus an object which has to be keyed by strings? And wasn't sure if the arrow function should be included by default or described in the readme to be added if desired?